### PR TITLE
Add link to sylius plus demo in homepage infobox

### PR DESCRIPTION
**Pull request by arti0090**

### CODE REVIEW

File: templates/_infoBox.html.twig [hunk 1]
```diff
@@ -15,6 +15,9 @@
                 Go to <a href="{{ 'https://master.demo.sylius.com/new-api/docs' }}">{{ 'https://master.demo.sylius.com/new-api/docs' }}</a><br><br>
             {% endif %}
 
+            <strong>How to access Sylius Plus demo?</strong><br>
+            Go to <a href="https://plus.demo.sylius.com">plus.demo.sylius.com</a><br><br>
```
Comment (CoderMaggie) [inline templates/_infoBox.html.twig:19]:
```suggestion
            Go to <a href="{{ 'https://plus.demo.sylius.com/' }}">{{ 'https://plus.demo.sylius.com/' }}</a><br><br>
```
to be consistent with other links? :)

**Comment by AdamKasp** [comment]
And please add PrintScreen with these changes

**Comment by arti0090** [comment]
Zrzut ekranu 2020-05-20 o 09 32 26
here you go @AdamKasp :)

**Comment by arti0090** [comment]
Zrzut ekranu 2020-05-20 o 09 46 47
Here You go @AdamKasp  :)

**Comment by CoderMaggie** [comment]
@arti0090 screenshot made before you fixed what I've suggested 👯

**Comment by AdamKasp** [comment]
please squash your commits 🗡️

File: templates/_infoBox.html.twig [hunk 2]
```diff
@@ -15,6 +15,9 @@
                 Go to <a href="{{ 'https://master.demo.sylius.com/new-api/docs' }}">{{ 'https://master.demo.sylius.com/new-api/docs' }}</a><br><br>
             {% endif %}
 
+            <strong>How to access Sylius Plus demo?</strong><br>
+            Go to <a href="{{ 'https://plus.demo.sylius.com' }}">{{ 'https://plus.demo.sylius.com' }}</a><br><br>
```
Comment (lchrusciel) [inline templates/_infoBox.html.twig:19]: I'm slightly wondering why we did it this way in the first place :)

**Comment by lchrusciel** [comment]
Thank you, @arti0090! :tada:

**Pull request merged**